pkg/node/kubelet.go

```diff
@@ -84,3 +87,74 @@ func StartKubeProxy(cfg *config.MicroshiftConfig) error {
 
 	return nil
 }
+
+const (
+	// proxy component name
+	component = "kubeproxy"
+)
+
+type ProxyOptions struct {
```

Comment:
Not that it matters in terms of functionality, but for consistency with other components, this struct should be called after the component itself.

Reply:
@oglok I noticed we do not follow the golang naming conventions for constants. Is it ok if I use golang standard `COMPONENT_KUBEPROXY = "kube-proxy"` instead of `componentKubeProxy = "kubeproxy"`